e2e/tests/features/sdk_website.feature

```diff
@@ -13,6 +13,23 @@ Feature: Protect and Track Demo Site
   Background:
     Given Project is protect-and-track-demo
 
+  @smoke @regression
+  Scenario: Send TDF3 attachment in BP
+    Given a TDF3 file: <attachment_name>
+    When I start to run testrail <case_id>
+    And I logged in gmail page as qavirtrumd11@cmk-test-org.com using a new browser with BP activated
+    And I composed an email to qavirtrumd12@cmk-test-org.com
+    And I turned on secure toggle
+    And I input attachment test as email subject in gmail page
+    And I input attachment test as email body in gmail page
+    And I drag and drop the file in gmail page
+    And I clicked send securely button in gmail page and wait for complete
+    Then Tooltip sample.tdf.html is visible for {attachment_name} in gmail page
```

Comment:
Short:
1. need to replace <attachment_name> {attachment_name} and <case_id> with actual values, remove `Examples` section
2. need to replace `sample.tdf.html` with one of the current available attachments: https://github.com/virtru/automated-test-scripts-v3/tree/develop/resources/attachments; otherwise need to add them into this [repo](https://github.com/virtru/automated-test-scripts-v3)

Long: 
https://cucumber.io/docs/gherkin/reference/

<img width="1128" alt="scenario_vs_outline" src="https://user-images.githubusercontent.com/25446794/81579221-e45f5400-9379-11ea-8e5f-9bb2053dd1fa.png">

Example:
https://github.com/virtru/automated-test-scripts-v3/blob/develop/secure_reader/tests/features/attachments.feature#L207

Reply:
I've checked in a commit changing to "short" form, removed the "Examples" tables. Thanks!